config/confighttp/README.md

```diff
@@ -34,6 +34,11 @@ README](../configtls/README.md).
 - [`disable_keep_alives`](https://golang.org/pkg/net/http/#Transport)
 - [`http2_read_idle_timeout`](https://pkg.go.dev/golang.org/x/net/http2#Transport)
 - [`http2_ping_timeout`](https://pkg.go.dev/golang.org/x/net/http2#Transport)
+- [`cookies`](https://pkg.go.dev/net/http#CookieJar)
+  - [`enabled`] if enabled, the client will store cookies from server responses and reuse them in subsequent requests.
+  - [`insecure`] if true, the client accepts setting cookies for any domain. This is useful for testing but is insecure:
+    it means that the HTTP server for foo.co.uk can set a cookie for bar.co.uk.
+    If false, the client will allow setting cookies based on the list provided by https://publicsuffix.org/
```

Comment:
Do we have an use case for this? If the only use case is for testing on the module itself, I would make the field private

Reply:
yes? I can see how this might be useful when troubleshooting.

Reply:
Unless you think it's essential, I would rather leave it out until someone asks for it explicitly so that we commit to a smaller public API